ethexe/signer/src/lib.rs

```diff
@@ -249,6 +250,12 @@ impl Signer {
         let local_public = PublicKey::from_bytes(public_key.serialize());
 
         let key_file = self.key_store.join(local_public.to_hex());
+        println!(
+            "Secret key: {:?}",
+            secret_key
+                .secret_bytes()
+                .to_hex_string(secp256k1::hashes::hex::Case::Lower)
```

Comment:
use secp256k1::hashes::hex::Case;

```suggestion
                .to_hex_string(Case::Lower)
```